insuree/apps.py

```diff
@@ -20,19 +21,21 @@
     "gql_mutation_create_insurees_perms": ["101102"],
     "gql_mutation_update_insurees_perms": ["101103"],
     "gql_mutation_delete_insurees_perms": ["101104"],
-    "insuree_photos_root_path":  os.path.abspath("./images/insurees"),
+    "insuree_photos_root_path": os.path.abspath("./images/insurees"),
     "excluded_insuree_chfids": ['999999999'],  # fake insurees (and bound families) used, for example, in 'funding'
     "renewal_photo_age_adult": 60,  # age (in months) of a picture due for renewal for adults
     "renewal_photo_age_child": 12,  # age (in months) of a picture due for renewal for children
-    "insuree_number_validator": None,  # Insuree number *function* that validates the insuree number
-    "insuree_number_length": None,  # Insuree number length to validate
+    "insuree_number_validator": None,  # Insuree number *function* that validates the insuree number for example
+                                       # IdentifierValidator().is_valid_resident_identifier
```

Comment:
I was wondering if config for chfid validation should be set up manually or automatically with if "insuree_as_worker": True then IdentifierValidator would be initialized. @dborowiecki @malinowskikam what do you think?